src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleManualTemplateFileUpload.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.ManualPayments;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities
+{
+    public class HandleManualTemplateFileUpload
+    {
+        private readonly IManualPaymentsService _manualPaymentsService;
+        private const int PaymentRecordsLimit = 2500;
+
+        public HandleManualTemplateFileUpload(IManualPaymentsService manualPaymentsService)
+        {
+            _manualPaymentsService = manualPaymentsService;
+        }
+
+        [FunctionName("HandleManualTemplateFileUpload")]
+        public async Task Run([BlobTrigger("employer-incentives-eups/PaymentData/{name}", Connection = "ConfigurationStorageConnectionString")]Stream blobStream, string name, ILogger log)
```

Comment:
How can we validate what is uploaded?  Any crap could be passed in in any order and we may end up loading invalid data?

Reply:
If it was a public facing service, I would agree, however it's an internal tool used for an exception process in the event that the payments API is unavailable, and will only be used by members of the team/DevOps in a test environment